addons/website/static/src/scss/website.scss

```diff
@@ -876,6 +876,12 @@ header {
         max-height: 80vh;
         overflow-y: auto;
         overflow-x: hidden; // Needed because of container in container having 0px padding... TODO improve
+
+        @include media-breakpoint-down(md) {
```

Comment:
this should be changed to
```scss
@include media-breakpoint-down(lg)
```
in the forward-port in V16. (see [this commit](https://github.com/odoo/odoo/commit/9b5c9484fe0bf2302f4269defebef18aff458dca))

Reply:
> this should be changed to
> 
> ```css-scss
> @include media-breakpoint-down(lg)
> ```
> 
> in the forward-port in V16. (see [this commit](https://github.com/odoo/odoo/commit/9b5c9484fe0bf2302f4269defebef18aff458dca))

done :heavy_check_mark: